src/modes/command/client/commands/hint.js

```diff
@@ -28,6 +28,10 @@ export const focusLink = background('findHints', {
   filter: '*',
   activate: 'focusLink'
 })
+export const yankLink = background('findHints', {
```

Comment:
Is "Yank Link" a command that copies anchor (<a>) links? If so, can you rename it to "Copy Link?"

Reply:
@sayash Please review comments.